src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -55,7 +55,15 @@ protected AbstractPlatformProxy(ILoggerAdapter logger)
         private IOtelInstrumentation InternalGetOtelInstrumentation()
         {
 #if SUPPORTS_OTEL
-            return new OtelInstrumentation();
+            try
+            {
+                return new OtelInstrumentation();
+            } catch (FileNotFoundException ex) 

```

Comment:
would it be safe to also include `TypeLoadException`?

Reply:
Hmm, not sure. This was the only exception that was thrown in this scenario. I would catch just this one, in case `TypeLoadException` is thrown in some unrelated scenarios?